nilearn/input_data/tests/test_fmriprep_confounds.py

```diff
@@ -0,0 +1,524 @@
+import re
+import numpy as np
+import pandas as pd
+from scipy.stats import pearsonr
+from sklearn.preprocessing import scale
+import pytest
+from nibabel import Nifti1Image
+from nilearn.input_data import NiftiMasker
+from nilearn.input_data import fmriprep_confounds
+from ..fmriprep_confounds import _sanitize_strategy
+
+from nilearn._utils.fmriprep_confounds import _to_camel_case
+
+from .utils import create_tmp_filepath, get_leagal_confound
+
+
+def _simu_img(tmp_path, demean):
```

Comment:
I think we should set a random state in these tests to avoid spurious test failures.

Reply:
I was looking into that - will do!